tf_big/python/tensor_test.py

```diff
@@ -277,6 +277,25 @@ def test_convert_to_tensor(self):
 
     assert y.dtype is tf.string
 
+  def test_limb_conversion(self):
+    context = tf_execution_context(False)
```

Comment:
Setting this to `True` will make tests pass, but if I'm setting it to false I get an error saying:
`ValueError: Shape must be rank 0 but is rank 1 for 'BigExportLimbs' (op: 'BigExportLimbs') with input shapes: [1], [1,2].`

So I'm guessing the problem is how I've registered the operation but not sure why this is a problem when the execution context is set to `False`.

Reply:
cc @mortendahl here, what are your thoughts?

Reply:
I suspect that shape inference is only invoked when running in graph mode (`False`) and not when running in eager mode (`True`). Believe the problem is that you're passing in `maxval=[20]` instead of `maxval=20`. I've pushed a commit for this change, but getting segfaults in return.